ayon_api/server_api.py

```diff
@@ -6132,7 +6131,8 @@ def get_repre_ids_by_context_filters(
 
         Context filters have defined structure. To define filter of nested
             subfield use dot '.' as delimiter (For example 'task.name').
-        Filter values can be regex filters. String or 're.Pattern' can be used.
+        Filter values can be regex filters. String or ``re.Pattern`` can
```

Comment:
Whats the reason this needs to be between two quotes? Was this intended to be `re.Pattern`?

Reply:
Thanks, I had only ever seen it with single quotes like:
```
`re.Pattern`
```